vueui/src/components/view/mcomputer/security/penalcontroller.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div>
+    <hr>
+    <h3>Remote Penal Mechs</h3>
+    <hr>
+    <div>
+      <div class="table">
+        <div class="header">
+          <div class="header-item">Pilot</div>
+          <div class="header-item">Mech Type</div>
+          <div class="header-item">Location</div>
+          <div class="header-item">Camera</div>
+          <div class="header-item">Lockdown</div>
+          <div class="header-item">End Connection</div>
+        </div>
+        <div class="mech" v-for="mech in mechs" :key="mech.ref">
+          <div class="item center">
+            <template v-if="mech.pilot"><vui-button :params="{message_pilot: mech.ref}">{{ mech.pilot }}</vui-button></template>
+            <template v-else><div class="red">n/a</div></template>
+          </div>
+          <div class="item center">{{ mech.name }}</div>
+          <div class="item center">{{ mech.location }}</div>
+          <div class="item center"><vui-button :class="{'selected': camera_selected == 1}" :params="{track_mech: mech.ref}" :disabled="mech.camera == 0">Track</vui-button></div>
+          <div class="item center"><vui-button :class="{'red': mech.lockdown == 1}" :params="{lockdown_mech: mech.ref}">Lockdown</vui-button></div>
+          <div class="item center"><vui-button :params="{terminate: mech.ref}" icon="eject">Terminate</vui-button></div>
+        </div>
+      </div>
```

Comment:
Please use `<table>`, `<tr>`, `<th>` and `<td>` instead of divs, as current classes conflict with some of global styles.